docs/overview/learning-path-beginner.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Start Here! A guide for beginners making Office Add-ins
+description:  A recommended path for beginners through the learning resources for Office Add-ins.
+ms.date: 02/19/2020
+ms.topic: conceptual
+ms.custom: scenarios:getting-started
+localization_priority: Priority
+---
+
+# Start Here! A guide for beginners making Office Add-ins
+
+Thank your for your interest in Office Add-ins, the Office extensions that run cross-platform, including Office on the web, Windows, and Mac. Like many people who are exploring a new framework, your first question might be **"Where do I start?"**. So, we created this recommended sequence of resources.
+
+## 1 Begin with fundamentals
+
+We know you're eager to start coding, but there are some things about Office Add-ins that you should read before you open your IDE or code editor.
+
+- [Office Add-ins Platform Overview](office-add-ins.md): Find out what Office Web Add-ins are and how they differ from older ways of extending Office, such as VSTO add-ins. (Spoiler alert: they're web apps embedded in Office and they run in Office on Mac as well as Windows.)
```

Comment:
I like the jovial tone here but would ask @lindalu-MSFT about whether spoiler alert is ok to use and translatable. 
```suggestion
- [Office Add-ins Platform Overview](office-add-ins.md): Find out what Office Web Add-ins are and how they differ from older ways of extending Office, such as VSTO add-ins. (Spoiler alert: they're cross-platform web apps embedded in Office). 
```